index.d.ts

```diff
@@ -749,7 +749,7 @@ declare namespace Moleculer {
 		static mergeSchemaUnknown(src: GenericObject, target: GenericObject): GenericObject;
 	}
 
-	type CheckRetryable = (err: Error) => boolean;
+	type CheckRetryable = (err: Errors.MoleculerError) => boolean;
```

Comment:
I don't necessarily see a problem with this, but I'm not 100% certain that all errors here are instances of `MoleculerError`.  @icebob Can you confirm for me if all errors passed to the retry check callback are always instances of `MoleculerError` or can they be other errors as well?

Reply:
No, it will be called with any Error that services can throw.